In-house patch to fix authenticaton 'failing' because of lack of cookie
support simply because the cookie was never set. This patch is unlikely
to be unsuitable for upstream, and should be removed when Django is
updated to 1.6.11 or greater, as the problem disappears with that
version.

--- django_openstack_auth-1.1.9/openstack_auth/forms.py.orig	2015-06-19 15:03:16.425066412 -0700
+++ django_openstack_auth-1.1.9/openstack_auth/forms.py	2015-06-19 15:05:16.267455199 -0700
@@ -71,6 +71,7 @@
         elif len(self.fields['region'].choices) > 1:
             self.fields['region'].initial = self.request.COOKIES.get(
                 'login_region')
+        self.request.session.set_test_cookie()

     @staticmethod
     def get_region_choices():
